### PR TITLE
added usign api mirror

### PR DESCRIPTION
**Pull request by edkesuma**

### CODE REVIEW

File: e-notary-server/routes/usign/authToken.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        }
+    }
+    var data = "";
+    var url = "https://api.usign.kr:18443/PDFsign/auth/verifyToken/";
+    axios.post(url, data, config)
+        .then((apiRes) => {
+            res.send(apiRes.data)
+        })
+        .catch((err) => {
+            console.log(err)
```
Comment (michael-shane) [inline e-notary-server/routes/usign/authToken.js]: if there is an error, check if it is internal server error or not, and still send a response with the appropriate information regarding the error

File: e-notary-server/routes/usign/createToken.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        }
+    }
+    var data = "";
+    var url = "https://api.usign.kr:18443/PDFsign/auth/createToken/";
+    axios.post(url, data, config)
+        .then((apiRes) => {
+            res.send(apiRes.data)
+        })
+        .catch((err) => {
+            console.log(err)
```
Comment (michael-shane) [inline e-notary-server/routes/usign/createToken.js]: Same issue as before regarding error response

File: e-notary-server/routes/usign/signDocument.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+router.post("/", (req, res, next) => {
+    var data = req.body;
+    var url = "https://api.usign.kr:18443/PDFsign/sign/";
+    axios.post(url, data)
+        .then((apiRes) => {
+            res.send(apiRes.data)
+        })
+        .catch((err) => {
+            console.log(err)
```
Comment (michael-shane) [inline e-notary-server/routes/usign/signDocument.js]: Same issue as before regarding error response

File: e-notary-server/routes/usign/authToken.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+var axios = require('axios');
```
Comment (michael-shane) [inline e-notary-server/routes/usign/authToken.js]: Use Const for imports

File: e-notary-server/routes/usign/createToken.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+var axios = require('axios');
```
Comment (michael-shane) [inline e-notary-server/routes/usign/createToken.js]: Use Const for imports

File: e-notary-server/routes/usign/signDocument.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+var axios = require('axios');
```
Comment (michael-shane) [inline e-notary-server/routes/usign/signDocument.js]: Use Const for imports

**Review by mshane911 — CHANGES_REQUESTED** [review]
Changes required, look at comments

**Review by mshane911 — APPROVED** [review]
LGTM!

**Pull request merged**